### ISSUE TITLE
Tests fail with python 3.6.4

### ISSUE THREAD

**Issue by tjboring** [PyGithub/PyGithub#740]
Just to be sure it wasn't a `PYTHONPATH` issue, I also re-ran the command below after setting it.

```
(pygithub) [2018-04-04 11:14 mbp] (master) PyGithub $ python -m github.tests
Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.6.4_4/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 183, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/usr/local/Cellar/python/3.6.4_4/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 109, in _get_module_details
    __import__(pkg_name)
  File "/Users/tboring/hearst/PyGithub/github/__init__.py", line 42, in <module>
    from MainClass import Github, GithubIntegration
ModuleNotFoundError: No module named 'MainClass'
```

**Comment by sfdye** [comment]
It's an known issue for python2/3 import compatibility, right now the work-around is to run `python setup.py test` if you are using python3.

**Comment by tjboring** [comment]
Ah, cool.  Thanks!

**Comment by labrys** [comment]
This would be fixed by #967
